R/fda-table_04.R

```diff
@@ -1,15 +1,13 @@
 #' FDA Table 4: Patient Disposition, Pooled Analyses
 #'
 #' @details
-#' * `df` must contain `USUBJID`, `RANDFL`, `ITTFL`,`PPROTFL`, `EOTSTT`, `DCTREAS`, `EOSSTT`, `DCSREAS`, and the
-#'   variables specified by `arm_var` and `saffl_var`.
-#' * If specified, `alt_counts_df` must contain `USUBJID` and the variable specified by `arm_var`.
+#' * `df` must contain `USUBJID`, `EOTSTT`, `DCTREAS`, `EOSSTT`, `DCSREAS` and the variables specified by `arm_var`
+#'   and `pop_var`.
+#' * If specified, `alt_counts_df` must contain `USUBJID`, and the variable specified by `arm_var` and `pop_var`.
 #' * Flag variables (i.e. `XXXFL`) are expected to have two levels: `"Y"` (true) and `"N"` (false). Missing values in
 #'   flag variables are treated as `"N"`.
 #' * Columns are split by arm. Overall population column is excluded by default (see `lbl_overall` argument).
-#' * Numbers in table "Patients Treated" section are the absolute numbers of patients and fraction of `N`.
 #' * All-zero rows are not removed by default (see `prune_0` argument).
-#' * Records with missing treatment start and/or end datetime are excluded from all calculations.
 #'
 #' @inheritParams argument_convention
 #'
```

Comment:
Please add parameter definitions for the two added variables here (`pop_vars` and `lbl_pop_vars`)

Reply:
Hi Emily, Thank you for your kind review. I have updated programs following your request.